src/Components/sharedComponents/ImageBuilderHeader.tsx

```diff
@@ -18,7 +18,13 @@ import {
 
 import './ImageBuilderHeader.scss';
 
-export const ImageBuilderHeader = () => {
+type ImageBuilderHeaderPropTypes = {
+  experimentalFlag?: string | true | undefined;
```

Comment:
New layout and buttons in experimental only. Seems to be working well! :sparkles: 

The only part that confuses me a bit is `string | true`, but that might be because I didn't look into flags much. Can we get both types?